api-schema/xcherry.yaml

```diff
@@ -184,7 +143,13 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/AsyncStateExecuteResponse'
-        default:
+        '406':
+          description: response of handling app database error
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AppDatabaseErrorHandling'
```

Comment:
should we also return the error message to xc server? so that in the future it could be shown as part of the process event history